examples/datadog/otel_collector_config.yaml

```diff
@@ -0,0 +1,58 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: "otel-agent"
+          scrape_interval: 10s
+          static_configs:
+            - targets: ["0.0.0.0:8888"]
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
```

Comment:
IMO the default ports should use the value `datadog.otelCollector.ports` present in the values.yaml

https://github.com/DataDog/helm-charts/pull/1429/files#diff-2fcec98016e714405bc936fd949012c2e72f23c20a1a94acf695a9cd3e29d75fR528-R535